modules/power/default.nix

```diff
@@ -0,0 +1,47 @@
+{ config, lib, ... }:
+
+with lib;
```

Comment:
```suggestion
```

Reply:
Why? I see this pattern is being used in the other modules.

Reply:
We're in the process of removing `with lib;` like `nixpkgs`

Reply:
I assume that the work to do so already is in progress, in a different branch/fork.

Otherwise it would be cleaner to make the change in one commit.

I'll remove `with lib;`.

Reply:
I don't think anyone's started writing it yet, however ideally preferably preventing new code from using `with lib;` will reduce the amount of work in the future refactor